tools/Performance/DynamoPerformanceTests/PerformanceTestConsoleApp.cs

```diff
@@ -1,48 +1,120 @@
 ﻿using System;
 using System.IO;
+using BenchmarkDotNet.Configs;
 using BenchmarkDotNet.Running;
+using NDesk.Options;
 
 namespace DynamoPerformanceTests
 {
     public class Program
     {
-        /// <summary>
-        /// This is the entrance of Dynamo Performance Console App
-        /// </summary>
-        /// <param name="args"></param>
+        private enum Command
+        {
+            Benchmark,
+            Debug,
+            InProcess,
+            Compare,
+        }
+
         public static void Main(string[] args)
         {
-            DirectoryInfo dir;
-            // Running with an input dir location:
-            // DynamoPerformanceTests.exe "C:\directory path\"
-            // 
+            var showHelp = false;
+
             if (args.Length <= 0)
             {
-                Console.WriteLine("Supply a path to a test directory containing DYN files");
+                Console.WriteLine("Please specify a command.");
+                return;
+            }
+
+            // Default arguments
+            IConfig config = PerformanceTestHelper.getFastReleaseConfig();
+            var testDirectory = "../../../graphs";
+            var baseResultsPath = "BenchmarkDotNet.Artifacts/results/DynamoPerformanceTests.PerformanceTestFramework-report.csv";
+            var newResultsPath = string.Empty;
+            var saveComparisonPath = string.Empty;
+
+            // Command line options
+            var opts = new OptionSet() {
+                { "g=|graphs=", "Path to Directory containing test graphs. Defaults to 'Dynamo/tools/Performance/DynamoPerformanceTests/graphs'", v => { testDirectory = v; } },
+                { "b=|base=", "Path to performance results file to use as comparison base. Defaults to 'BenchmarkDotNet.Artifacts/results/DynamoPerformanceTests.PerformanceTestFramework-report.csv'", v => { baseResultsPath = v; }},
```

Comment:
I think the defaults path now got switched after my new changes
![image](https://user-images.githubusercontent.com/3942418/55772982-a1d39580-5a5c-11e9-826b-db916f11c0b8.png)

Reply:
Yeah, good idea. For both the baseline and the new results, maybe we should point to a folder, and concatenate all the csvs in that folder. That way we get both the Model and View results.